### ISSUE TITLE
Error: e.loc is undefined (font shorthand destructuring)

### ISSUE THREAD

**Issue by bartveneman** [projectwallace/css-analyzer#453]
The analyzer fails on this CSS (which is incorrect, but it should not throw an error):

```css
test {
  font: 14px "Inter Var", sans-serif, 700;
}
```

Reproduction https://stackblitz.com/edit/stackblitz-starters-jt1fpabn?file=index.js
